packages/wdio-cli/src/templates/wdio.conf.tpl.ejs

```diff
@@ -184,15 +176,30 @@ exports.config = {
     // Options to be passed to Mocha.
     // See the full list at http://mochajs.org/
     mochaOpts: {
-        ui: 'bdd',
+        <% if(answers.isUsingTypeScript) {
+        %>// TypeScript setup
+        require: 'ts-node/register',
+        compilers: ['tsconfig-paths/register'],
```

Comment:
`compilers` is deprecated, let's use `require` instead.
`require` accepts an array (not sure if string works at all)